connect/templates/generator-maven-plugin/README.md

```diff
@@ -22,20 +20,17 @@
 
 | Name                     |                | Description                                                      | Possible Values           | Default Value                                                    |
 |--------------------------|----------------|------------------------------------------------------------------|---------------------------|------------------------------------------------------------------|
+| `project`                | **(required)** | The Maven project.                                               |                           | ${project}                                                       |
```

Comment:
Why is project a required field? What does it do?

Reply:
It defines for which maven project the plugin is executed. It is required but there is also a default value (${project}) which is taken if you don't specify anything.